<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>贷款类别</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>贷款方式</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>商业贷款</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>公积金贷款</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>组合型贷款</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>根据面积、单价计算</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>根据贷款总额计算</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>87</height>
     </rect>
    </property>
    <property name="plainText">
     <string>房屋单价


房屋面积</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>元/平米


平米</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>87</height>
     </rect>
    </property>
    <property name="plainText">
     <string>商贷贷款


公积金贷款</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>万元


万元</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>按揭成数


按揭年数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3成</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1年(12期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2年(24期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3年(36期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4年(48期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5年(60期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6年(72期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7年(84期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8年(96期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9年(108期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10年(120期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11年(132期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12年(144期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13年(156期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14年(168期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15年(180期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16年(192期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17年(204期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18年(216期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19年(228期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20年(240期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21年(252期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22年(264期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23年(276期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24年(288期)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25年(300期)</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>商贷利率


公积金利率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>%


%</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>还款方式</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>等额本息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>等额本金</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;	使用说明书&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;若选择商业贷款或公积金贷款 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  请先选择贷款方式&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  1.根据面积、单价计算&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    输入房屋单价、房屋面积及对应的贷款利率，并选择按揭成数、按揭年数、还款方式&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  2.根据贷款总额&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    输入对应贷款总额、利率，并选择按揭年数，还款方式&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;若选择组合型贷款&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  只需输入商贷贷款总额、公积金贷款总额及其对应的利率，再选择还款方式即可计算&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>计算结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>乞丐版房贷计算器</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
